lib/packages/docutils/rstgen.nim

```diff
@@ -57,7 +57,9 @@ type
     options*: RstParseOptions
     findFile*: FindFileHandler
     msgHandler*: MsgHandler
-    filename*: string
+    destBaseDir*: AbsoluteDir   ## output directory, initialized by docgen.nim
```

Comment:
examples always help, and would immediately clarify the subtlety, eg:
/pathto/htmldocs for destBaseDir; vs /pathto/htmldocs/sub1/sub2/strutils.html

Reply:
ok now that you've renamed as outDir it's kind of clear

Reply:
Also, I have a little example in the if/else block that I've added in this PR.